### DISCUSSION TITLE
Is it possible to Player control option like "play , pause , forward and backward buttons" ?

### DISCUSSION THREAD

**Discussion by mahalax** [alexandra-c/leaflet-tracking-marker#22] [Q&A]
Is it possible to add Player control option like "play , pause , forward and backward buttons" ?

**Comment by alexandra-c** [comment]
Hello, 
This would be a great addition to the library indeed, it can be done for sure. I don't know when I would have time to do this tho'... I'm gonna try and find time and give it a try.
